apps/fabric-website/src/pages/GetStarted/GetStartedPage.module.scss

```diff
@@ -109,9 +112,9 @@
 }
 
 @media screen and (min-width: $uhf-screen-min-mobile) {
-  img {
-    margin-bottom: 20px;
-  }
+  // img {
```

Comment:
Can remove this if it's not longer in use.

Reply:
Thanks! Removed.